styles/module/portfolio-preview.css

```diff
@@ -0,0 +1,67 @@
+/* Portfolio Preview */
+
+.project-preview {
+  background-size: 100%;
+  background-repeat: none;
+  cursor: pointer;
+  border: 1px solid #ffffff;
+}
+
+.project-preview section{
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  background: rgba(61, 211, 178, 0.8);
+  display: block;
+  padding: 10px;
+}
+.project-preview section p {
+  margin: 5px 0;
+}
+
+.project-details{
```

Comment:
separate preview and detail view into their own module. You'll be applying additional CSS to the detailed preview that will have no affect on the preview.